syntax = "proto3";

package vr_events;

option optimize_for = LITE_RUNTIME;
option java_package = "com.actein.vr_events";
option java_outer_classname = "VrBoothInfoProtos";

message VrBoothInfo
{
  int32 id = 1;
}
